README.md

```diff
@@ -1,35 +1,35 @@
-# Your Project Name
+# Spotify Player
```

Comment:
Project is too simple it does not use at least 3 labs and 3 peripherals. Also you cannot steal some code from the internet for the project.